docs/10-WebServices.md

```diff
@@ -19,21 +19,43 @@ Configure modules in `api.suite.yml`:
 ``` yaml
 class_name: ApiTester
 modules:
-    config:
-		- REST:
-		    url: http://serviceapp/api/v1/
-		    depends: PhpBrowser
+    enabled:
+				- REST:
+				    url: http://serviceapp/api/v1/
+				    depends: PhpBrowser
 ```
```

Comment:
But the example is wrong, it should be:
```
class_name: ApiTester
modules:
  enabled:
    - REST
  config:
    REST:
      url: http://joomla.box/repos/redshopb2b/tests/joomla-cms3/
      depends: PhpBrowser
```

Otherwise you get: `[Symfony\Component\Yaml\Exception\ParseException]  `